codebase/superdarn/src.bin/tk/tool/make_fit.1.19/make_fit.c

```diff
@@ -71,7 +71,6 @@ int main(int argc,char *argv[]) {
   unsigned char version=0;
 
   unsigned char vb=0;
-  unsigned char old_elev=0;
```

Comment:
This and some other lines were not intended to be removed by the latest commit right? Just need to revert them.

Reply:
Yes I remerged the make_fit file back in but I need to make sure other files were not over written